spec/spec_helper.rb

```diff
@@ -1,5 +1,17 @@
+ENV['RACK_ENV'] = 'test'
```

Comment:
this is what the rubric recommends for the spec_helper:

RSpec.configure do |config|

  config.include Capybara::DSL # good
  config.include FactoryGirl::Syntax::Methods # sure
  config.include TestHelpers # good
end